app-kotlin/build.gradle

```diff
@@ -36,9 +36,9 @@ android {
 dependencies {
   implementation project(':library')
   implementation deps.kotlinstdlib
-  implementation deps.appcompatv7
-  implementation deps.constraintlayout
-  implementation deps.design
+  implementation 'androidx.appcompat:appcompat:1.1.0-alpha01'
+  implementation 'androidx.constraintlayout:constraintlayout:2.0.0-alpha3'
+  implementation 'com.google.android.material:material:1.1.0-alpha02'
```

Comment:
This dependencies should be added in the top-level `build.gradle` file and referenced here as other dependencies. I organized this project in such way and it should be consistent across all modules.